UnityProject/Assets/Scripts/Managers/SubsystemBehaviourQueueInit.cs

```diff
@@ -25,6 +25,11 @@ private static void ClearSubsystems()
 
 		public static void Queue(SubsystemBehaviour behaviour)
 		{
+			if (Instance.behaviours.Contains(behaviour))
```

Comment:
anticipating to what Bod might say, do we know now why it can be added twice?

Reply:
It's not getting added twice to this system, but I just added it there just incase.
What is actually happening is that its adding it to two systems, the new one and the old one; the old one cannot be removed for the time being because the entire game implodes if you try using the new system.. even if the new system does the same exact thing.